mscore/resourceManager.cpp

```diff
@@ -538,7 +538,7 @@ static std::vector<PluginPackageLink> getAttachments(const QString& html_raw, vo
                   const QStringList suffixes = { "qml","zip","rar","7z" };
                   if (!suffixes.contains(QFileInfo(match.captured(1)).suffix().toLower()))
                         continue;
-                  PluginPackageLink link = { {ATTACHMENT},{match.captured(1)}};
+                  PluginPackageLink link{ {ATTACHMENT},{match.captured(1)}};
```

Comment:
I believe you need `PluginPackageLink link{ {ATTACHMENT, match.captured(1)};};` here in in those other 3 places?

Reply:
OK, I'll try this again.

Reply:
Unfortunately both `PluginPackageLink link{ {ATTACHMENT,match.captured(1)} };` and `PluginPackageLink link{ {ATTACHMENT,match.captured(1)}; };` failed.